### DISCUSSION TITLE
Error installing update 4.2.0

### DISCUSSION THREAD

**Discussion by Gavin-Perry** [earlephilhower/arduino-pico#2595] [General]
Here is the log when I tried to install the newest update on Windows 10 Enterprise:

Tool rp2040:pqt-gcc@4.0.1-8ec9d6f already installed
Tool rp2040:pqt-gcc-riscv@4.0.1-8ec9d6f already installed
Tool rp2040:pqt-mklittlefs@4.0.1-8ec9d6f already installed
Tool rp2040:pqt-pioasm@4.0.1-8ec9d6f already installed
Tool rp2040:pqt-python3@1.0.1-base-3a57aed-1 already installed
Tool rp2040:pqt-openocd@4.0.1-8ec9d6f already installed
Tool rp2040:pqt-picotool@4.0.1-8ec9d6f already installed
Downloading packages
rp2040:rp2040@4.2.0
Replacing platform rp2040:rp2040@4.1.1 with rp2040:rp2040@4.2.0
Uninstalling rp2040:rp2040@4.1.1
Running pre_uninstall script.
Platform rp2040:rp2040@4.1.1 uninstalled
Uninstalling rp2040:pqt-gcc@4.0.1-8ec9d6f, tool is no more required
Uninstalling rp2040:pqt-gcc-riscv@4.0.1-8ec9d6f, tool is no more required
Uninstalling rp2040:pqt-mklittlefs@4.0.1-8ec9d6f, tool is no more required
Uninstalling rp2040:pqt-openocd@4.0.1-8ec9d6f, tool is no more required
Uninstalling rp2040:pqt-picotool@4.0.1-8ec9d6f, tool is no more required
Uninstalling rp2040:pqt-pioasm@4.0.1-8ec9d6f, tool is no more required
Uninstalling rp2040:pqt-python3@1.0.1-base-3a57aed-1, tool is no more required
Configuring platform.
Platform rp2040:rp2040@4.2.0 installed
**Failed to install platform: 'rp2040:rp2040:4.2.0'.
Error: 14 UNAVAILABLE: read ECONNRESET**

So where am I now? Do I need to back up to 4.1.1 (HOW?) or will it work OK with this error?

**Comment by earlephilhower** [comment]
Looks like you're having network issues.  `ECONNRESET` points to something between you and GitHub resetting the TCP connection.  Please give it another go, possibly on a different network (if laptop)...
